examples/fabric-bridge-app/linux/include/DeviceManager.h

```diff
@@ -66,3 +66,12 @@ class DeviceManager
     chip::EndpointId mFirstDynamicEndpointId;
     Device * mDevices[CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT + 1];
 };
+
+/**
+ * Instance getter for the global DeviceManager.
+ *
+ * Callers have to externally synchronize usage of this function.
+ *
+ * @return The global DeviceManager. Assume never null.
+ */
+DeviceManager * GetDeviceManager();
```

Comment:
What parts of the code do we expect to use this global?

Wondering if we could have a path to inject the device manager instead of creating singletons. I think for one-off maybe globals are convenient, however if we expect integrations of this code into other applications, then not using singletons seems better and more testable.

Reply:
DeviceManager is used to manage the list of devices to be synced. It is utilized by all entities where the Fabric Sync protocol is implemented, such as the RPC Client, RPC Server, and all relevant ChipCommands.

Its role is very similar to PlatformMgr, which is why I have adopted a similar access pattern to PlatformMgr.